# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1380487642 14400
#      Sun Sep 29 16:47:22 2013 -0400
# Node ID 9c994995ddb00b1e007a6b17f439243080d3bf21
# Parent  21656a94966107d1d7f8f3a77281c1bad26500c2
doc: Fix reference to __unimplemented__

* grammar.txi: Fix reference to __unimplemented__ internal function.

diff --git a/doc/interpreter/grammar.txi b/doc/interpreter/grammar.txi
--- a/doc/interpreter/grammar.txi
+++ b/doc/interpreter/grammar.txi
@@ -73,14 +73,14 @@ In addition, there are three non-specifi
 periodic evaluation.  @code{remove_input_event_hook} will stop a user function
 from being evaluated periodically.
 
 @DOCSTRING(add_input_event_hook)
 
 @DOCSTRING(remove_input_event_hook)
 
 Finally, when the parser cannot identify an input token it calls a particular
-function to handle this.  By default, this is the function
-@qcode{"unimplemented"} which makes suggestions about possible Octave
+function to handle this.  By default, this is the internal function
+@qcode{"__unimplemented__"} which makes suggestions about possible Octave
 substitutes for @sc{matlab} functions.
 
 @DOCSTRING(missing_function_hook)
 
